src/utils/update/trust.model.fetch.ts

```diff
@@ -319,7 +319,12 @@ const mapTrusteeType = (trusteeTypeId: string): RoleWithinTrustType | undefined
       case "5002":
         return RoleWithinTrustType.BENEFICIARY;
       case "5001":
-        // Type 5001 is not a valid type for an individuual or corporate trustee
+        /*
+          Type 5001 is not a valid type for an individuual or corporate trustee. Instances of individual or corporate
```

Comment:
Minor, but some spelling mistakes - 'individuual', 'rusteeTypeId', 'shoud'.

Thanks for adding the comment!

Reply:
Thanks. Have installed a spell checker into my IDE on the back of this.